pyccel/parser/semantic.py

```diff
@@ -1094,7 +1094,7 @@ def _handle_function(self, expr, func, args, is_method = False):
         func : FunctionDef instance, Interface instance or PyccelInternalFunction type
                The function being called.
 
-        args : tuple
+        args : list
                The arguments passed to the function.
```

Comment:
Do we care if `args` is a list or a tuple? I suppose it just needs to be iterable, as we unpack it below.

Reply:
when i was executing the code inside this method, i checked the type of `args` and it was `list` so i thought I'd update the doc, but iterable makes as you mentioned the unpacking

Reply:
@Farouk-Echaref please mention if and where my comment was addressed.

Reply:
@yguclu fixed

Reply:
solved in [ed6e77c](https://github.com/pyccel/pyccel/pull/1727/commits/ed6e77c4d083b01b3de93bbb8ce87f3a379e6f6d)